# SceneState : represents a state of objects in a scene through predicates

Header header
Predicate[] predicates
